src/components/button.js

```diff
@@ -81,4 +83,20 @@ class Button extends Component {
   }
 }
 
+Button.propTypes = {
+  children: PropTypes.string.isRequired,
+  editing: PropTypes.bool.isRequired,
+  onClick: PropTypes.func,
+  opacity: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  style: PropTypes.shape({
+    fontSize: PropTypes.number,
+  }),
+  className: PropTypes.string,
+  onButtonDelete: PropTypes.func,
+  onChange: PropTypes.func,
+  zoomed: PropTypes.bool,
+  nodeId: guid,
+  ee: PropTypes.func, //I'm not exactly sure what this is supposed to be. Not yet defined in app.
```

Comment:
Kill it.

It's cruft leftover from when I was using [this package](https://npm.im/nee)